src/dowel/csv_output.py

```diff
@@ -42,12 +42,27 @@ def record(self, data, prefix=''):
                 self._writer.writeheader()
 
             if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
-                           'CsvOutput keys: {}. '
-                           'TabularInput keys: {}. '
-                           'Did you change key sets after your first '
-                           'logger.log(TabularInput)?'.format(
-                               set(self._fieldnames), set(to_csv.keys())))
+                                
+                # update new field name
```

Comment:
I think the logic here is correct, but it should probably use plurals (e.g. name*s* and field*s*), since multiple new fields can (and often do) appear together.

Reply:
I don't quite get why should we use plurals, what is the names in here refer to? The set update function do support multiple new fields appear together.